docs/Backtracing.rst

```diff
@@ -330,3 +338,184 @@ large number of frames in a much smaller space than would otherwise be possible.
+| Field             | Value                                                  |
++===================+========================================================+
+| timestamp         | An ISO-8601 formatted timestamp, as a string.          |
++-------------------+--------------------------------------------------------+
+| kind              | The string ``crashReport``.                            |
++-------------------+--------------------------------------------------------+
+| description       | A textual description of the crash or runtime failure. |
++-------------------+--------------------------------------------------------+
+| faultAddress      | The fault address associated with the crash.           |
++-------------------+--------------------------------------------------------+
+| platform          | A string describing the platform; the first token      |
+|                   | identifies the platform itself and is followed by      |
+|                   | platform specific version information.                 |
+|                   |                                                        |
+|                   | e.g. "macOS 13.0 (22A380)",                            |
+|                   |      "linux (Ubuntu 22.04.5 LTS)"                      |
++-------------------+--------------------------------------------------------+
+| architecture      | The name of the processor architecture for this crash. |
++-------------------+--------------------------------------------------------+
+
+These will be followed by some or all of the following, according to the
+backtracer settings:
+
++-------------------+--------------------------------------------------------+
+| Field             | Value                                                  |
++===================+========================================================+
+| threads           | An array of thread record, one for each thread (if the |
+|                   | backtracer is set to give backtraces for all threads). |
++-------------------+--------------------------------------------------------+
+| crashedThread     | A single thread record for the crashing thread (if the |
+|                   | backtracer is set to give a backtrace only for the     |
+|                   | crashed thread).                                       |
++-------------------+--------------------------------------------------------+
+| registers         | A dictionary containing the register contents on the   |
+|                   | crashed thread (if set to give registers for only the  |
+|                   | crashed thread).                                       |
+|                   |                                                        |
+|                   | The dictionary is keyed by architecture specific       |
+|                   | register name; values are given as hexadecimal         |
+|                   | strings.                                               |
```

Comment:
Are these prefixed with `0x`? I assumed not reading here, but it looks like the code does emit the prefix.